rotkehlchen/tests/unit/globaldb/test_asset_updates.py

```diff
@@ -512,6 +513,16 @@ def test_conflict_updates(assets_updater: AssetsUpdater, globaldb: GlobalDBHandl
     """Test that the logic doesn't add duplicates for assets that were inserted twice
     in the globaldb assets updates
     """
+    with globaldb.conn.write_ctx() as write_cursor:
+        assert write_cursor.execute(
+            'SELECT COUNT(*) FROM underlying_tokens_list WHERE parent_token_entry=?;',
+            ('eip155:42161/erc20:0xA5EDBDD9646f8dFF606d7448e414884C7d905dCA',),
+        ).fetchone()[0] > 0
```

Comment:
since you know exactly how many set them in a variable and check that the amount didn't change

Reply:
Also add in the docstring that you test that remote errors don't alter the underlying tokens/multiasset mappinggs and this is a regression test for `_force_remote_asset`